src/main/java/uk/gov/hmcts/reform/sscs/services/mapper/CaseDataBuilder.java

```diff
@@ -253,8 +253,10 @@ Subscriptions buildSubscriptions(final Optional<Parties> representativeParty, St
     }
 
     private String validateMobile(Optional<Parties> representativeParty, String appealCaseRefNum) {
-        if (representativeParty.isPresent()) {
-            String mobileNumber = representativeParty.get().getPhone2();
+        if (representativeParty.isPresent()
+            && representativeParty.get().getMobile() != null
+            && !representativeParty.get().getMobile().isEmpty()) {
```

Comment:
cannot we use here: StringUtils.isNotBlank(representativeParty.get().getMobile())